### PR TITLE
Add btn-extra-pixelize class

### PR DESCRIPTION
**Pull request by loo41**

### CODE REVIEW

File: scss/elements/buttons.scss [hunk 1]
```diff
@@ -73,3 +129,28 @@
...
+  @include btn-extra-pixelize-style($base-color, #fff, #e7e7e7, #adafbc);
+
+  $types:
+    "primary" #fff #92cc41 #76c442 #4aa52e, 
+    "success" #fff #209cee #108de0 #006bb3,
+    "warning" $base-color #f7d51d #f2c409 #e59400, 
+    "error" #fff #e76e55 #ce372b #8c2022;
+
+  @each $type in $types {
+    &.#{nth($type, 1)} {
```
Comment (BcRikko) [inline scss/elements/buttons.scss]: please add the prefix of `is-` 🙏 

for example, `.btn.is-rounded.is-primary`, `.btn.is-rounded.is-success`...

```suggestion
    &.is-#{nth($type, 1)} {
```

File: scss/elements/buttons.scss [hunk 2]
```diff
@@ -14,6 +14,62 @@
...
 
+@mixin btn-extra-pixelize-style($color, $background, $hover-background, $shadow) {
+  color: $color;
+  background-color: $background;
+
+  &:hover,
+  &:focus {
+    background-color: $hover-background;
+    outline: 0;
+    box-shadow:
```
Comment (BcRikko) [inline scss/elements/buttons.scss]: please add a comment to disable automatic formatting by prettier. 📝 

```suggestion
    // prettier-ignore
    box-shadow: 
```

File: scss/elements/buttons.scss [hunk 3]
```diff
@@ -14,6 +14,62 @@
...
+      0 8px #212529,
+      -4px 0 $hover-background, 
+      inset -2px 0 $shadow, 
+      -4px -4px #212529, 
+      -8px 0 #212529,
+      -4px 4px #212529;
+  }
+
+  &:active {
+    box-shadow:
```
Comment (BcRikko) [inline scss/elements/buttons.scss]: same here 🙏 
```suggestion
    // prettier-ignore
    box-shadow: 
```

File: scss/elements/buttons.scss [hunk 4]
```diff
@@ -14,6 +14,62 @@
...
+      0 4px $hover-background, 
+      4px 4px #212529, 
+      0 8px #212529, 
+      -4px 0 $shadow,
+      -4px -4px #212529, 
+      -8px 0 #212529, 
+      -4px 4px #212529;
+  }
+
+  box-shadow:
```
Comment (BcRikko) [inline scss/elements/buttons.scss]: same here 🙏 
```suggestion
  // prettier-ignore
  box-shadow: 
```

File: scss/elements/buttons.scss [hunk 5]
```diff
@@ -73,3 +129,28 @@
...
+// Btn dot
+.btn-extra-pixelize {
+  $border-size: 4px;
+
+  box-sizing: content-box;
+  display: inline-block;
+  padding: 6px 8px;
+  margin: $border-size;
+  cursor: pointer;
+  border: none;
```
Comment (BcRikko) [inline scss/elements/buttons.scss]: please overwrite only the necessary properties 🙏 

```scss
.btn {
  // default styles (default properties)
  $border-size: 4px;

  position: relative;
  display: inline-block;
  padding: 10px 12px;
  margin: $border-size;
  text-align: center;
  vertical-align: middle;
  cursor: pointer;
  user-select: none;
 
  ...

  // overwrite: like this↓
  &.is-rounded {
    box-sizing: content-box;
    padding: 6px 8px;
    border: none;
  }
}
```

**Comment by loo41** [comment]
ok

**Comment by loo41** [comment]
Please look at it

**Comment by loo41** [comment]
Sorry, it's too careless

**Comment by evexoio** [comment]
@loo41 I just added .btn-extra-pixelize to story book. Somehow it's not working, is it an issue from my end?

I made sure to check out to your branch first.

extra-pixelize

**Comment by evexoio** [comment]
@loo41 I just added .btn-extra-pixelize to story book. Somehow it's not working, is it an issue from my end?

I made sure to check out to your branch first.

extra-pixelize

**Comment by loo41** [comment]
Instead of adding. btn-extra-pixelize, it's. btn. is-rounded. is-primary

**Comment by loo41** [comment]
Instead of adding. btn-extra-pixelize, it's. btn. is-rounded. is-primary @evexoio

**Comment by trezy** [comment]
Hey @loo41, make sure to merge the latest version of the `develop` branch so that our CI will run the required checks for this PR. 😉

**Comment by loo41** [comment]
🌹

**Comment by BcRikko** [comment]
LGTM 🎉

**Comment by evexoio** [comment]
@loo41 can you please squash your commits?

this is so the commit history of the repo don't get polluted by commit's trying things out 👍

**Comment by loo41** [comment]
Sorry, the operation is wrong. I will close this PR

**Pull request closed without merging**